README.md

```diff
@@ -34,15 +34,28 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
-2. Name two big differences between ```display: block;``` and ```display: inline;```.
+A: I would tell them that semantic HTML gives your code and the look of your application meaning. It also seperates your code into their own groups and makes thing organized.
+
+2. Name two big differences between `display: block;` and `display: inline;`.
+
+A: 1: display: block gives that element its own line. It takes up the space that it wants. inline is displayed inside the current block on the same line.
```

Comment:
which technically means blocks will expand to fill 100% of the parent containers width
✔️